src/index.html

```diff
@@ -11,7 +11,121 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset_style">
+        <legend>Personal information</legend>
+
+        <label>
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+
+        <label class="form_style">
+          Name:
+          <input type="text" name="Name">
+        </label>
+
+        <label class="form_style">
+          How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form_style">
+          Full date of birth:
+          <input type="date">
+        </label>
+
+        <label class="form_style">
+          I accept the term of the agreement
+          <input type="checkbox">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset_style">
+        <legend>Registration</legend>
+
+        <label>
+          E-mail:
+          <input type="email" placeholder="email@example.com">
+        </label>
+
+        <label class="form_style">
+          Password:
+          <input
+            type="password"
+            minlength="1"
+            maxlength="10"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset_style">
+        <legend>An interesting fact about you!</legend>
+
+        Do you love cats?
+        <label>
+          <input type="radio" name="agree">
+          Yes
+        </label>
+
+        <label>
+          <input type="radio" name="agree">
+          No
+        </label>
```

Comment:
this name is not clear

imagine that you are developer that processing this form on backend, you didn't see design of this form and you should update particular data in database depending on received information, and you receiving such object

```
{
  agree: 'yes',
}
```

will you understand what exactly should you update? 🙂

Reply:
Please use more precise names

Reply:
And add value to radio buttons